specification/schemas/PaymentIdentity.json

```diff
@@ -0,0 +1,89 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/MinimalPaymentIdentity"
+    },
+    {
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "provider": {
+              "type": "string",
+              "example": "Stripe"
+            },
+            "credentials": {
+              "type": "object",
+              "example": {
+                "token": "FHzWRJl5guoU6IIH"
+              }
+            },
+            "credit_card_supported": {
+              "type": "boolean",
+              "default": false,
+              "example": true
+            },
+            "debit_card_supported": {
+              "type": "boolean",
+              "default": false,
+              "example": false
+            },
+            "sepa_supported": {
+              "type": "boolean",
+              "default": false,
+              "example": false
+            },
+            "ideal_supported": {
+              "type": "boolean",
+              "default": false,
+              "example": false
+            },
+            "paypal_supported": {
+              "type": "boolean",
+              "default": false,
+              "example": false
+            },
+            "created_at": {
+              "type": "number",
+              "readOnly": true,
+              "example": 1504801719,
+              "description": "Unix timestamp when this payment-identities resource was created."
+            },
+            "updated_at": {
+              "type": "number",
+              "readOnly": true,
+              "example": 1504801719,
+              "description": "Unix timestamp when this payment-identities resource was last updated."
```

Comment:
For other resources with `created_at` and `updated_at` we use `"$ref": "#/components/schemas/Timestamp"`